app/frontend/styles/theme/components/button.ts

```diff
@@ -61,5 +61,14 @@ export const Button = {
       },
       _disabled: { ...disabledStyles, bg: "transparent" },
     },
+    link: {
+      color: "text.link",
+      fontWeight: "normal",
+      _hover: {
+        textDecoration: "underline",
+        _disabled: { textDecoration: "none" },
+      },
+      _disabled: { ...disabledStyles, bg: "inherit" },
+    },
```

Comment:
I think this is the same as the `tertiary` button

Reply:
The color is different. Should I:
- Manually specify the color on the component?
- change the tertiary component?
- ignore the color?